hibernate-reactive-core/src/test/java/org/hibernate/reactive/configuration/JdbcUrlParserTest.java

```diff
@@ -33,6 +36,9 @@
 
 	private static final String DEFAULT_DB = "hreactDB";
 
+	@Rule
+	public DatabaseSelectionRule rule = DatabaseSelectionRule.skipTestsFor( H2 );
```

Comment:
Our Vertx supported DB's all use the `DefaultSqlClientPoolConfiguration.connectOptions()` method. `SqlConnectOptions` requires a host and port, so for H2, it would mean coding around these checks.

In `H2SqlClientPool`  I'm using a local `SqlConnectOptions` object, but I don't need to.  Since this class is isolated and uses vertx-jdbc-client `JDBCConnectOptions` anyway I think it might be better to not use SqlConnectOptions for H2.

Reply:
Since this test uses `DefaultSqlClientPool` and `DefaultSqlClientPoolConfiguration().connectOptions( uri )` which are specific to non-H2 Db's I think it still makes sense to skip H2 dbtype

Reply:
We can update the test so that it works for both, though. Anyway, ok, let's fix everything else first